src/main/java/uk/co/setech/easybook/model/BaseEntity.java

```diff
@@ -0,0 +1,23 @@
+package uk.co.setech.easybook.model;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.MappedSuperclass;
+import lombok.Data;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.LocalDateTime;
+
+@Data
+@MappedSuperclass
+public abstract class BaseEntity {
+    @Id
+    @GeneratedValue//(strategy = GenerationType.IDENTITY)
```

Comment:
WE NEED TO SWITCH TO IDENTITY

Reply:
I can see you done it already, why not make it for everything?